docs/updates/2019-03.md

```diff
@@ -0,0 +1,47 @@
+# March 2019
+
+Welcome to the March 2019 Qlik Core update. We're highlighting a couple of important changes and improvements which we
+hope you will like.
+
+## Engine security enhancement
+
+Because of a security concern and to follow best-practices, the Qlik Associative Engine Docker image is changing its
+user and group from `nobody:nobody` to `1910:1910`. You can read more about using an unique UID/GID here:
+[https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#user](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#user).
+
+If you are on Mac or Linux and using Docker volume mounts, you may need to change the owner of the volume files. You can
+do that with the following command:
+
+```bash
+chown -R 1910:1910 /YOUR-MOUNT
+```
+
+## Introducing hamus.js
+
+We have grown tired of typing the same code every time we create a new react.js solution, so we decided to consolidate
+reusable code into a new utility library called [hamus.js](https://github.com/qlik-oss/hamus.js). The repo utilizes
+the new React hook API and offers a set of hooks which simplifies communication with the Qlik Associative Engine.
+
+The repo comes with an example to get you started. Check it out at [hamus.js](https://github.com/qlik-oss/hamus.js).
```

Comment:
Perhaps also add something like `Initially it contains a set of hooks to simplify the usage of enigma.js models and layouts, as well as rendering picasso.js charts based on hypercubes. We will add more hooks as we identify reusable code in projects, and from feedback. If you have ideas, feel free to share them as issues in the GitHub repository below.`